# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1522446309 14400
#      Fri Mar 30 17:45:09 2018 -0400
# Branch stable
# Node ID b940e457e11fb9d50294bd16c5f4228a0dec6c62
# Parent  74ece2b45b4f7359e27a8374f47d381d8319bc53
maint: Update gnulib sources ahead of 4.4 release.

diff --git a/.hgsub b/.hgsub
--- a/.hgsub
+++ b/.hgsub
@@ -1,1 +1,1 @@
-gnulib = http://hg.octave.org/gnulib
+gnulib = http://hg-new.octave.org/gnulib
diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,1 +1,1 @@
-733fbf6b39bc505d2717f25e8b7d114aeaa94098 gnulib
+791f51373ba1b80c8ae3b7a9d2509d8a827a3e25 gnulib
